src/main/java/teammates/ui/webapi/GetSessionAccessLogsAction.java

```diff
@@ -0,0 +1,36 @@
+package teammates.ui.webapi;
+
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.EntityNotFoundException;
+import teammates.common.util.Const;
+
+/**
+ * Action: gets the session access logs of feedback sessions of a course.
+ */
+public class GetSessionAccessLogsAction extends Action {
+    @Override
+    AuthType getMinAuthLevel() {
+        return AuthType.PUBLIC;
```

Comment:
Missed this earlier, but is it better for it to be `AuthType.LOGGED_IN`? Can't think of a situation where the instructor can get the logs without being logged in first